src/app/modules/main/wallet_section/networks/model.nim

```diff
@@ -170,6 +170,7 @@ QtObject:
 
     return (chainIds, enable)
 
+  # TODO: To be removed once all qml calls to this method are removed. Normally the formatting methods should live in the qml project.
```

Comment:
is there a task to link to?

Reply:
Nope. I'll create one!

Reply:
https://github.com/status-im/status-desktop/issues/15499